tests/camel-integration-test/src/main/java/org/apache/karaf/camel/itests/CamelSuppliedRouteLauncher.java

```diff
@@ -61,17 +63,22 @@ public void deactivate() {
     }
 
     private void loadSuppliers() {
-        String property = System.getProperty(CAMEL_KARAF_INTEGRATION_TEST_ROUTE_SUPPLIERS_PROPERTY);
-        if (property == null) {
-            return;
+        final String property = System.getProperty(CAMEL_KARAF_INTEGRATION_TEST_ROUTE_SUPPLIERS_PROPERTY);
+        if (property != null) {
+            suppliers.addAll(List.of(property.split(",")));
+        }
+        String ignoreProperty = System.getProperty(CAMEL_KARAF_INTEGRATION_TEST_ROUTE_IGNORE_SUPPLIERS_PROPERTY);
+        if (ignoreProperty != null) {
+            ignoreSuppliers = Boolean.parseBoolean(ignoreProperty);
         }
```

Comment:
```suggestion
        String ignoreProperty = System.getProperty(CAMEL_KARAF_INTEGRATION_TEST_ROUTE_IGNORE_SUPPLIERS_PROPERTY);
        if (ignoreProperty != null) {
            ignoreSuppliers = true;
            return;
        }
        String property = System.getProperty(CAMEL_KARAF_INTEGRATION_TEST_ROUTE_SUPPLIERS_PROPERTY);
        if (property == null) {
            return;
        suppliers.addAll(List.of(property.split(",")));    
```
Again they are exclusives

Reply:
Done